docs/content/docs/dev/datastream/fault-tolerance/state.md

```diff
@@ -429,6 +429,9 @@ will lead to compatibility failure and `StateMigrationException`.
 
 - The TTL configuration is not part of check- or savepoints but rather a way of how Flink treats it in the currently running job.
 
+- It is not recommended to restore checkpoint state with adjusting the ttl from a short value to a long value,
```

Comment:
IIUC, It just affects RocksDBStateBackend instead of HashMapStateBackend, right ?

Reply:
As described in [jira](https://issues.apache.org/jira/browse/FLINK-32953), RocksDBStateBackend may exist data correctness problem. But both RocksDBStateBackend and HashMapStateBackend have issue with expired data becoming visible after restore. 
So, I think this notes should take effect for both RocksDBStateBackend and HashMapStateBackend.
WDYT?

Reply:
Thanks for the explaination. LGTM.